src/model.js

```diff
@@ -256,11 +260,21 @@ export default class MediaPlayerObject {
   }
 
   volumeUp(e) {
-    this.callService(e, 'volume_up');
+    if (this.supportsVolumeSet) {
+      this.callService(e, 'volume_set', {
+        entity_id: this.config.entity,
+        volume_level: this.vol + this.config.volume_step / 100,
```

Comment:
Need to make sure the `volume_level` prop is between 0..1 or it will throw an error and not execute the volume change, could use a clamp here, similar to 
https://github.com/kalkih/mini-media-player/blob/fa1427db8be8b7f099d01df4b7feb8df954b68a7/src/components/progress.js#L131

```js
Math.max(0, Math.min(volume, 1));
```

Reply:
https://github.com/kalkih/mini-media-player/pull/308/commits/1fdd8dc7f7c67952518820a3f8576914e45a5526